specification/columns/commitmentdiscountcategory.md

```diff
@@ -32,6 +32,7 @@ Allowed values:
 |:--------|:-------------------------------------------------------------------------|
 | Spend   | Commitment-based discounts that require a predetermined amount of spend. |
 | Usage   | Commitment-based discounts that require a predetermined amount of usage. |
+| Other   | Commitment-based discounts that do not fall into another category.       |
```

Comment:
Is this just to make sure the column remains applicable for the future?  I don't know of any commitment models that fall outside of "Usage" or "Spend" offhand.

Reply:
Correct, this is just about covering a new scenario that doesn't fit into the existing buckets. There are no other known commitment discount types today.